docs/content/getting-started/setup-openfga.mdx

```diff
@@ -21,9 +21,9 @@ If you want to run OpenFGA locally as a Docker container, follow these steps:
 
 1. [Install Docker](https://docs.docker.com/get-docker/) if Docker was not already installed.
 2. Run `docker pull openfga/openfga` to get the latest docker image.
-3. Run `docker run -p 8080:8080 openfga/openfga run`.
+3. Run `docker run -p 8080:8080 -p 8081:8081 -p 3000:3000 openfga/openfga run`.
 
-This will start an HTTP server and gRPC server with the default configuration options.
+This will start an HTTP server and gRPC server with the default configuration options. Port 8080 is used to serve the HTTP API, 8081 is used to serve the grpc API, and 3000 is used for serve the [Playground](#playground).
```

Comment:
```suggestion
This will start an HTTP server and gRPC server with the default configuration options. Port 8080 is used to serve the HTTP API, 8081 is used to serve the gRPC API, and 3000 is used for the [Playground](#playground).
```